core/io/pwrite_spec.rb

```diff
@@ -28,16 +28,42 @@
       @file.pread(6, 0).should == "foobar"
     end
 
+    it "calls #to_s on the object to be written" do
+      data = mock("to_s")
```

Comment:
as far as parameter names are used in the tests - it seems reasonable to use specific name `object` instead of generic `data`:
```suggestion
      object = mock("to_s")
```

Reply:
Changed.